1.20/Dockerfile.fedora

```diff
@@ -62,12 +62,12 @@ COPY ./root/ /
 # 2) for working root-less container with UID=1001, which does not have
 #    to have GID=0
 # 3) for default use-case, that is running container directly on operating system,
-#    with default UID and GID (1001:1)
+#    with default UID and GID (1001:0)
 # Supported combinations of UID:GID are thus following:
 # UID=1001 && GID=0
 # UID=<any>&& GID=0
 # UID=1001 && GID=<any>
-RUN sed -i -f ${NGINX_APP_ROOT}/nginxconf-fed.sed ${NGINX_CONF_PATH} && \
```

Comment:
why nginxconf-fed.sed is not used if it already exists in `root/opt/app-root`. Or should we delete it?